src/Shared/Components/DeploymentConfigDiff/DeploymentConfigDiffRadioSelect.tsx

```diff
@@ -0,0 +1,85 @@
+import RadioGroup from '@Common/RadioGroup'
+import RadioGroupItem from '@Common/RadioGroupItem'
+import { ComponentSizeType } from '@Shared/constants'
+
+import { Button, ButtonStyleType, ButtonVariantType } from '../Button'
+import { Icon } from '../Icon'
+import { Popover, usePopover, UsePopoverProps } from '../Popover'
+import { DeploymentConfigDiffRadioSelectConfig } from './DeploymentConfigDiff.types'
+
+import './DeploymentConfigDiff.scss'
+
+const DeploymentConfigDiffRadioSelect = ({
+    radioSelectConfig,
+    position = 'bottom',
+}: {
+    radioSelectConfig: DeploymentConfigDiffRadioSelectConfig
+    position?: UsePopoverProps['position']
+}) => {
+    const { open, closePopover, overlayProps, popoverProps, triggerProps } = usePopover({
+        id: 'deployment-config',
+        width: 300,
+        position,
+        variant: 'overlay',
+    })
+
+    const { triggerElementTitle, radioGroupConfig } = radioSelectConfig
+
+    const renderTriggerElement = () => (
+        <div className="flex dc__content-space dc__gap-8">
+            <span className="fs-13 cn-9 fw-6 lh-20 dc__truncate">{triggerElementTitle}</span>
+            <Icon name="ic-caret-left" color={null} size={20} rotateBy={open ? 90 : -90} />
+        </div>
+    )
+
+    return (
+        <Popover
+            open={open}
+            triggerElement={renderTriggerElement()}
+            buttonProps={null}
+            popoverProps={popoverProps}
+            overlayProps={overlayProps}
+            triggerProps={triggerProps}
+        >
+            <div className="flexbox-col">
```

Comment:
```suggestion
            <div ref={scrollableRef} className="flexbox-col dc__overflow-auto">
```

`scrollableRef` get this from `usePopover` hook